build.sh

```diff
@@ -59,13 +59,47 @@ popd
 if [[ "${HOST_TRIPLE}" != "x86_64-pc-windows-msvc" ]] ; then
     git clone --single-branch --branch solana-tools-v1.44 https://github.com/anza-xyz/newlib.git
     echo "$( cd newlib && git rev-parse HEAD )  https://github.com/anza-xyz/newlib.git" >> version.md
-    mkdir -p newlib_build
-    mkdir -p newlib_install
-    pushd newlib_build
+
+    mkdir -p newlib_build_v0
+    mkdir -p newlib_v0
+    pushd newlib_build_v0
+    CC="${OUT_DIR}/rust/build/${HOST_TRIPLE}/llvm/bin/clang" \
+      AR="${OUT_DIR}/rust/build/${HOST_TRIPLE}/llvm/bin/llvm-ar" \
+      RANLIB="${OUT_DIR}/rust/build/${HOST_TRIPLE}/llvm/bin/llvm-ranlib" \
+      ../newlib/newlib/configure --target=sbf-solana-solana --host=sbf-solana --build="${HOST_TRIPLE}" --prefix="${OUT_DIR}/newlib_v0"
+    make install
+    popd
+
+    mkdir -p newlib_build_v1
+    mkdir -p newlib_v1
+    pushd newlib_build_v1
+    CFLAGS="-mcpu=v1" \
+    CC="${OUT_DIR}/rust/build/${HOST_TRIPLE}/llvm/bin/clang" \
+      AR="${OUT_DIR}/rust/build/${HOST_TRIPLE}/llvm/bin/llvm-ar" \
+      RANLIB="${OUT_DIR}/rust/build/${HOST_TRIPLE}/llvm/bin/llvm-ranlib" \
+      ../newlib/newlib/configure --target=sbf-solana-solana --host=sbf-solana --build="${HOST_TRIPLE}" --prefix="${OUT_DIR}/newlib_v1"
+    make install
+    popd
+
+    mkdir -p newlib_build_v2
+    mkdir -p newlib_v2
+    pushd newlib_build_v2
+    CFLAGS="-mcpu=v2" \
+    CC="${OUT_DIR}/rust/build/${HOST_TRIPLE}/llvm/bin/clang" \
+      AR="${OUT_DIR}/rust/build/${HOST_TRIPLE}/llvm/bin/llvm-ar" \
+      RANLIB="${OUT_DIR}/rust/build/${HOST_TRIPLE}/llvm/bin/llvm-ranlib" \
+      ../newlib/newlib/configure --target=sbf-solana-solana --host=sbf-solana --build="${HOST_TRIPLE}" --prefix="${OUT_DIR}/newlib_v2"
+    make install
+    popd
+
+    mkdir -p newlib_build_v3
+    mkdir -p newlib_v3
+    pushd newlib_build_v3
+    CFLAGS="-mcpu=v3" \
```

Comment:
This is a bunch of repeated code. Let me get this PR in to match the version I published out there, and then I'll create a function for briefness in another PR.